docs/source/tutorial/introduction.rst

```diff
@@ -0,0 +1,438 @@
+########################
+Introduction to retworkx
+########################
+
+This guide serves as an introduction to working with retworkx. If you're a
+current or past `NetworkX <https:://networkx.org>`__ user who is looking at
+using retworkx as a replacement for NetworkX, you can also refer to
+:ref:`networkx` for a detailed guide comparing using retworkx with NetworkX.
+
+Creating a Graph
+================
+
+To create an empty undirected graph :math:`G` with no nodes and no edges you
+can run the following code:
+
+.. jupyter-execute::
+
+    import retworkx as rx
+    G = rx.PyGraph()
+
+A :class:`~retworkx.PyGraph` is an undirected collection of nodes (vertices)
+along with pairs of nodes (called edges, links, etc). Nodes and edges in
+retworkx are assigned any Python object (e.g. a numeric value, a string, an
+image, an xml object, another graph, a custom node object, etc) for a data
+payload (also referred to as a weight in the API and documentation). Nodes
+and edges are uniquely identified an integer ``index`` which is stable for
```

Comment:
typo: identified by

Reply:
Fixed in: https://github.com/Qiskit/retworkx/pull/537/commits/f7e4421111f967d25a0b2303bb34e5e110fdd8cc